### PR TITLE
Improve role explanation on invite modal

### PR DESCRIPTION
**Pull request by malted**
image

### CODE REVIEW

File: app/views/events/_form_invite.erb [hunk 1]
```diff
@@ -16,12 +16,24 @@
...
+          <li>Spend money</li>
+          <li>Not change the organization's settings</li>
+        </ul>
+      </p>
+
+    </div>
   <% end %>
 
-  <% admin_tool("mr1 flex g1 w-fit") do %>
+  <% admin_tools("mb2 flex g1 w-fit") do %>
```
Comment (sarthaktexas) [inline app/views/events/_form_invite.erb]: we're using the renamed `admin_tool` method now
```suggestion
  <% admin_tool("mb2 flex g1 w-fit") do %>
```

File: app/views/events/_organization_position_invite_pending.html.erb [hunk 2]
```diff
@@ -32,17 +32,18 @@
...
+    <div class="inline" data-controller="menu" data-action="click->menu#toggle click@document->menu#close keydown@document->menu#keydown" data-menu-target="toggle">
+      <% can_change_role = Flipper.enabled?(:user_permissions_2024_03_09, @event) && organizer_signed_in?(as: :manager) %>
+
+      <span class="h6 bold info <%= "pointer" if can_change_role %>"><%= invite.role %></span>
+
+      <% if can_change_role %>
+        <div data-menu-target="content" class="menu__content menu__content--2 menu__content--compact">
+          <% OrganizerPositionInvite.roles.keys.each do |role| %>
+            <%= link_to change_position_role_organizer_position_invite_path(id: invite.id, to: role), data: { turbo_method: "post", turbo_frame: "_top" } do %>
+              <%= role %><%= " ✔" if invite.role == role %>
```
Comment (sarthaktexas) [inline app/views/events/_organization_position_invite_pending.html.erb]: Visually it shows selected, but I don't think this is accessible. Can you add an `aria-current` for screenreaders?

File: app/views/events/_form_invite.erb [hunk 3]
```diff
@@ -16,12 +16,24 @@
...
+      <div class="muted mb1">
+        <p class="m0">Someone invited as a <span class="bold info">member</span> can</p>
+        <ul class="m0">
+          <li>Create cards</li>
+          <li>Spend money</li>
+          <li>Not change the organization's settings</li>
+        </ul>
+      </p>
+
+    </div>
```
Comment (sarthaktexas) [inline app/views/events/_form_invite.erb]: This copy is a good start. I think making them a consistent format would be better. I specifically mean choosing between "Someone invited..." vs "Inviting someone..." What if we changed which one was displayed based on what was selected? (Alpine.js can do this easily)

**Review by sarthaktexas — DISMISSED** [review]

File: app/views/events/_organization_position_invite_pending.html.erb (hunk 2 above)
Comment (malted) [reply app/views/events/_organization_position_invite_pending.html.erb]: Done on both invites and pending invites.

File: app/views/events/_form_invite.erb (hunk 3 above)
Comment (malted) [reply app/views/events/_form_invite.erb]: It did change in response to selection before! @maxwofford suggested to keep it static - I agree with him.

Comment (malted) [reply app/views/events/_form_invite.erb]: Changed the copy!

Comment (sarthaktexas) [reply app/views/events/_form_invite.erb]: Ah, sorry then!

**Comment by sarthaktexas** [comment]
How is your ERB Lint failing? It passes for me

**Comment by garyhtou** [comment]
> How is your ERB Lint failing? It passes for me

Maybe there's a closing tag missing somewhere. it currently looks like this
image

File: app/views/events/_organization_position_invite.html.erb [hunk 4]
```diff
@@ -9,15 +9,16 @@
       </div>
       <div class="flex flex-column justify-center">
         <p class="h2 bold m0"><%= position.user.name %></p>
```
Comment (garyhtou) [inline app/views/events/_organization_position_invite.html.erb]: It looks like some User Permission UI is leaking outside of the feature flag

**Review by garyhtou — CHANGES_REQUESTED** [review]

File: app/views/events/_organization_position_invite.html.erb (hunk 4 above)
Comment (malted) [reply app/views/events/_organization_position_invite.html.erb]: This was fixed in https://github.com/hackclub/hcb/pull/5483

**Comment by malted** [comment]
@garyhtou Ok now I'm really confused. It looks normal for me.

File: app/views/events/_organization_position_invite.html.erb (hunk 4 above)
Comment (garyhtou) [reply app/views/events/_organization_position_invite.html.erb]: It _was_ un-fixed by this PR!!
image

I've added a new commit to re-fix it

File: app/views/events/_form_invite.erb [hunk 5]
```diff
@@ -16,12 +16,23 @@
...
+      </p>
+
+      <div class="muted mb1">
+        <p class="m0">Someone invited as a <span class="bold info">member</span> can</p>
+        <ul class="m0">
+          <li>Create cards</li>
+          <li>Spend money</li>
+          <li>Not change the organization's settings</li>
+        </ul>
+      </div>
```
Comment (garyhtou) [inline app/views/events/_form_invite.erb]: After merging this PR, let's work on the copy for this

File: app/views/events/_organization_position_invite_pending.html.erb [hunk 6]
```diff
@@ -32,17 +32,18 @@
...
-      as a
-      <div class="inline" data-controller="menu" data-action="click->menu#toggle click@document->menu#close keydown@document->menu#keydown" data-menu-target="toggle">
-        <span class="h6 bold info <%= "pointer" if organizer_signed_in?(as: :manager) %>"><%= invite.role %></span>
-        <% if organizer_signed_in?(as: :manager) %>
-          <div data-menu-target="content" class="menu__content menu__content--2 menu__content--compact">
-            <% OrganizerPositionInvite.roles.keys.each do |role| %>
-              <%= link_to change_position_role_organizer_position_invite_path(id: invite.id, to: role), data: { turbo_method: "post", turbo_frame: "_top" } do %>
-                <%= role %><%= " ✔" if invite.role == role %>
-              <% end %>
+    Invited <%= relative_timestamp(invite.created_at) %> as a
```
Comment (garyhtou) [inline app/views/events/_organization_position_invite_pending.html.erb]: After merging, let's come back to this card to do a copy edit/visual design. Here's what it currently looks like

image

**Review by garyhtou — APPROVED** [review]
thanks!

**Pull request merged**